test_schemas/en/test_textfield_typeahead.json

```diff
@@ -0,0 +1,65 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Other input fields",
+    "theme": "default",
+    "description": "A questionnaire to test textfields",
+    "messages": {
+        "NUMBER_TOO_LARGE": "Number is too large",
+        "NUMBER_TOO_SMALL": "Number cannot be less than zero",
+        "INVALID_NUMBER": "Please enter an integer"
+    },
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "country-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "country-answer",
+                                        "label": "What is your country of birth?",
+                                        "mandatory": false,
+                                        "url": "https://gist.githubusercontent.com/rmccar/c123023fa6bd1b137d7f960c3ffa1fed/raw/368a3ea741f72c62c735c319ff7e33e3c1bfdc53/country-of-birth.json",
```

Comment:
I don't think a personal gist should be committed?

Reply:
This is where the data currently is.

Reply:
Is it being hosted in a CDN soon?. The feature should have a test before it is merged,

Reply:
Once the work has been scheduled to do so.

The only way of getting it into a cdn seems to be a pipeline and the existing one for the design system seems to go via aws. Spoke with @ajmaddaford on how we should approach this and we decided that setting up an appropriate pipeline was a little out of the scope of this card. Hence continuing to use the gist.

Reply:
So we should still write a functional test without the data being hosted in CDN then.

Reply:
I don't think there's a nice way of doing this - it would require custom endpoints within runner in order to modify csp policies.

Reply:
Ah right -_-

Reply:
Possibly, but no-one realised that until I'd done the work to integrate. So we can either add the schema now and make the test part of any cdn work, or just throw this away. We chose the former.

Reply:
Spoke with @ajmaddaford and agreed we can progress this by adding an integration test to ensure that the text field is rendered with the correct URL property and have confidence that the typeahead itself is tested in the design system.